README.md

```diff
@@ -33,7 +33,7 @@ Topics:
 * `cd` into the _friends_ folder and type `yarn add redux react-redux redux-thunk redux-logger axios react-router-dom` which will install the needed dependencies.
```

Comment:
You forgot to add the POST friends functionality.

MVP: POST Login, GET operation, and Add Friend Post.
STRETCH: PUT request to update the list and to delete it.